Directory.Build.props

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Build Settings">
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
We have projects targeting net4x, so the implicit LangVersion doesn't always help